grid/websocket_client.py

```diff
@@ -58,13 +58,17 @@ def on_client_result(args):
             # Tell the wait_for_client_event to clear up and continue execution
             self.wait_for_client_event = False
 
+        @self.__sio.on("/connect-node")
+        def connect_node(msg):
+            print(msg)
```

Comment:
Remove print message or change it to something like: "Connect node message: %s" % msg

Reply:
Solved at 7aeb21e